### PR TITLE
Add core typography tokens for Android

### PR DESCRIPTION
**Pull request by janogarcia**
Core typography tokens for Android. Most useful for generating Figma Styles.

Please note that it doesn't include composite tokens for monospace fonts and uppercase text yet.

### CODE REVIEW

File: tokens/platform-android.json [hunk 1]
```diff
@@ -1 +1,338 @@
...
+          "type": "letterSpacing"
+        },
+        "md": {
+          "value": "0.25px",
+          "type": "letterSpacing"
+        },
+        "lg": {
+          "value": "0.25px",
+          "type": "letterSpacing"
+        }
```
Comment (gsouquet) [inline tokens/platform-android.json:136]: Both iOS and Web use percentages for letter spacing. Is that expected to have pixel values here?

And for my own understanding, what's the rationale behind that?

File: assets/android/kotlin/CpdDark.kt [hunk 2]
```diff
@@ -162,6 +162,35 @@ object DarkDesignTokens {
   val cpdColorGray300 = Color(0xffe2e6ea)
   val cpdColorGray200 = Color(0xffeef1f3)
   val cpdColorGray100 = Color(0xfff7f8f9)
+  val cpdFontLetterSpacingHeadingXl = 0
```
Comment (gsouquet) [inline assets/android/kotlin/CpdDark.kt:165]: The output, once again looks wrong. I'll do the same exercise with the Android team

**Review by gsouquet — APPROVED** [review]
Otherwise looks good to me

File: tokens/platform-android.json (hunk 1 above)
Comment (janogarcia) [reply tokens/platform-android.json:136]: I added the rationale to the internal doc on transforming typography tokens, but just realized I haven't pushed it yet. Let me do it now.

Comment (janogarcia) [reply tokens/platform-android.json:136]: Actually, I pushed it yesterday. https://github.com/vector-im/compound-internal/blob/internal-docs/docs/typography.md#android

**Pull request merged**